src/encoded/search_views.py

```diff
@@ -64,6 +65,7 @@ def includeme(config):
     'Lab'
 ]
 
+TARGET_MATRIX_TITLE = 'TargetMatrix'
```

Comment:
Don't want to start putting constants here. Probably just type TargetMatrix into your view.